std_msgs/Header header
geometry_msgs/Pose type_in_pose
---
uint8 SUCCEEDED=0
uint8 FAILED=1
uint8 result_status
geometry_msgs/Pose pose_on_blackboard
